api/base/serializers.py

```diff
@@ -135,6 +137,90 @@ def to_representation(self, value):
         return {'links': {self.link_type: {'href': url, 'meta': meta}}}
 
 
+class JSONAPIHyperlinkedRelatedField(ser.HyperlinkedRelatedField):
```

Comment:
Hmm. Just internal refactoring? As long as it doesn't change any of the behavior that comments depend on, I'm good with that.

(just want to avoid QA testing the same repsonse formatting twice)